content/applications/inventory_and_mrp/inventory/management/products/strategies.rst

```diff
@@ -1,75 +1,200 @@
+  as a sales order, purchase order, or manufacturing order.
+- :guilabel:`Min Quantity`: The minimum amount of product that should be available. When inventory
+  levels goes below this number, the replenishment is triggered.
+- :guilabel:`Max Quantity`: The amount of product that should be available after replenishing the
+  product.
+- :guilabel:`Multiple Quantity`: If the product should be ordered in specific quantities, enter the
+  number that should be ordered. For example, if the :guilabel:`Multiple Quantity` is set to 5, and
+  only 3 are needed, 5 products will be replenished.
+- :guilabel:`To Order`: The amount of product that is currently needed, and will be ordered, if the
+  :guilabel:`Order Once` or :guilabel:`automate Orders` button is clicked.
+- :guilabel:`UoM`: The unit of measure used to acquire the product.
+- :guilabel:`Company`: The company the product is acquired for.
+
+By default, the quantity in the :guilabel:`To Order` field is the quantity required to reach the set
+:guilabel:`Max Quantity`. However, the :guilabel:`To Order` quantity can be adjusted by clicking on
+the field and, changing the value, and clicking :guilabel:`Save`. To replenish a product manually,
+click :guilabel:`Order Once`.
+
+To automate a replenishment (from the :guilabel:`Replenishment` page) click :guilabel:`Automate
+Orders` on the right side of the line, with a :guilabel:`🔄 (circular arrow)` icon.
+
+When this button is clicked, Odoo will automatically generate a draft :abbr:`PO (purchase order)`/
+:abbr:`MO (manufacturing order)` every time the forecasted stock level falls below the set
+:guilabel:`Min Quantity` of the reordering rule.
+
+On the :guilabel:`Replenishment` page, a reordering rule or manual replenishment can be temporarily
+deactivated for a given period, by clicking the :guilabel:`🔕 (snooze)` icon on the far-right of the
+line.
+
+.. image:: strategies/reordering-rule-snooze-settings.png
+   :align: center
+   :alt: Snooze options to turn off notifications for reordering for a period of time.
 
-Then, click on your product to access the related product form and, on
-the "Inventory submenu", do not forget to select a supplier.
+A :abbr:`PO (purchase order)` or :abbr:`MO (manufacturing order)` created by a manual replenishment
+has a :guilabel:`Replenishment Report` as the source document. A :abbr:`PO (purchase order)` or
+:abbr:`MO (manufacturing order)` created by an automated reordering rule will have the
+:abbr:`SO (sales order(s))` reference number(s) that triggered the rule as the source document.
 
-.. image:: strategies/strategies02.png
+.. image:: strategies/rfq-source-document.png
    :align: center
+   :alt: Quote request list shows which quotes are directly from the replenishment report.
 
-.. tip::
-   Don't forget to select the right product type. A consumable can not be stocked and will thus not
-   be accounted for in the stock valuation.
 
-Make to Order
--------------
+Make to order (MTO)
+===================
```

Comment:
I believe this heading was already used above? Perhaps another title/heading could help differentiate it from the previous one?